build.sh

```diff
@@ -1,72 +1,35 @@
 #!/bin/bash
 set -e
 
-CACHE_DIR=cache
+if [ "$CI" = "true" ]
+then
+	# Running on CI. Use a cache folder off the home directory.
+	CACHE_DIR="$HOME/cache"
```

Comment:
Since the caching step was removed from the GHA workflow, we don't need this case logic anymore, do we? You could just use `Fiji.app` without the word "cache" involved at all. (If it's not actually going to be cached, I would avoid using the term "cache")